src/poliastro/plotting.py

```diff
@@ -42,7 +42,7 @@ class OrbitPlotter(object):
     :py:meth:`set_frame` to set the frame before plotting.
 
     """
-    def __init__(self, ax=None, num_points=100):
+    def __init__(self, ax=None, num_points=100, bgcolor=(1,1,1), linewidth=1.5, markersize=6, audivision=6):
```

Comment:
pep8: space after commas

And also: why the name of `audivision`? Seems weird to me.